Tests/Features/resource_api.feature

```diff
@@ -0,0 +1,32 @@
+Feature: Request Resources from the REST API
+    In order to retrieve data from the resource webservice
+    As a webservice user
+    I need to be able to query the webservice
+
+    Background:
+        Given the test application has the following configuration:
+        """
+        cmf_resource:
+            repository:
+                doctrine_phpcr_odm:
+                    testrepo:
+                        basepath: /cmf/articles
+        """
+
+
+    Scenario: Retrieve a resource
+        Given there exists a "Article" document at "/cmf/articles/foo":
+            | title | Article 1 |
+            | body | This is my article |
+        Then I send a GET request to "/api/testrepo/foo"
+        Then open response
+        And the response code should be 200
```

Comment:
Actually the `Then open response` is a debug line I added to open the response in my web browser :)